canonical-web-and-design/ubuntu.com
# Add login and username to the navigation header

## Done
Add login and username to the navigation header

## QA
- Go to the demo
- See there is an action "Sign in" in the header
- Click it and see you return to the same page signed in
- Check your name in the top right
- Click your name and see your drop-down of local services
- Check different viewports (Small, medium and large) __There is no drop-down on small screens__

## Issue / Card
Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/9288
Fixes #9289 

## Screenshots
Small
![image](https://user-images.githubusercontent.com/1413534/113875785-05f5d500-97af-11eb-8c3f-bf9a174e685f.png)

Medium
![image](https://user-images.githubusercontent.com/1413534/113875853-13ab5a80-97af-11eb-98fc-f7634c8a6022.png)

Large
![image](https://user-images.githubusercontent.com/1413534/113875923-232aa380-97af-11eb-8d7e-d941cdd5c992.png)

Menu active
![image](https://user-images.githubusercontent.com/1413534/113876002-38073700-97af-11eb-86d0-74a94ed3a5d7.png)
